### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by allun26**

### CODE REVIEW

**Comment by mosaaleb** [comment]
Reviewing

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Hi @allun26, @vieiramanda11 :wave:

### Well done team :clap:. The UI is clean. Your code is organized and easy to read. You have handled most of the requirements, yet there is some requirements needs be met before moving on.


#### Navaigation bar:
- [x] It should use the `<nav>` HTML tag
- [x] It should have seven separate containers (`<div>` is recommended) -- (nav links as seen on the smashing magazine webpage)
- [x] Containers should be positioned using the `float` or `flex` display properties


#### Main content
- [x]  All content should be wrapped with `<main>` HTML tags
- [x] Every child of main should be a `<section>` HTML tag pair
- [x] Every child of section should be an `<article>` HTML tag pair

#### Half width containers (latest articles)
- [x] Each container should have an image in the top left corner with the title of an article under it.

#### General
- [x] We don't need to enable rubocop in `.stickler.yml` as it is the linter for ruby code. We just need to enable style lint. See [this](https://github.com/microverseinc/linters-config/tree/master/css) for more information

- [x] Let's fill the readme file with its relevant information.


Please sumbit a new code review once you finish these requirements.

### Happy coding :computer:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+                <h3>1500 smashing members</h3>
+                <p>Lorem ipsum dolor sit amet, ex eos eligendi dissentias. Lorem ipsum dolor sit amet, ex eos eligendi dissentias.</p>
+        </div>
+    </div>
+    <div class="seeAllLatestArticles">
+        <p>SEE ALL LATEST ARTICLES</p>
+    </div>
+    <div class="footer">
+        <h4>BROWSE ALL SMASHING MAGAZINE TOPICS</h4>
+        <ul>
```
Comment (mosaaleb) [inline index.html:177]: Let's use `<footer>` here for better semantic tags.

**Comment by samgaco** [comment]
### Hi @allun26 and @vieiramanda11 :vulcan_salute: 

### Good job implementing the changes pointed out by the previous reviewer. And the project is looking excellent :100: 

Still some things to improve:

* **Add images in the top right corners of all the articles in this section** (As you did with the first 4 half-width containers right after the navbar)

Screenshot_2019-07-25_19-47-16

* Try to make the footer look more like the original (similar alignment to the center and structure, etc). Use display flex or floats for arranging the unordered lists.

### Your project
footertu

### Original
footeell

* Add a project description to the **readme file** (what you tried to achieve, etc)

* Correct the errors in the index file given to you by https://validator.w3.org and take a look at the warnings as well.

### That's it. When you are done with these improvements, commit the changes and submit back for another code review. Regards! :muscle:

**Comment by samgaco** [comment]
### **Hi  @allun26  and @vieiramanda11 :hand:** 

### Congratulations :tada:  the project is approved. You can merge to the master branch and submit the project completion form.

As a **suggestion**, try to get rid of the warnings given by https://validator.w3.org

### Regards and happy coding! :muscle:

**Pull request merged**